Makefile

```diff
@@ -39,9 +39,9 @@ priv:
 	mkdir -p $(MIX_APP_PATH)/priv/
 
 priv/neutron_nif.so: ./c_src/neutron_nif.c
-	  $(CC) $^ -shared -lei $(PLATFORM_OPTIONS) -fPIC -O3 -finline-functions -Wunused -Wall -Wpointer-arith -Wcast-align -Wcast-qual $(ERL_CFLAGS) $(ERTS_CFLAGS) $(ERL_LDFLAGS) -dynamiclib -pedantic -I/usr/local/ssl/include -L/usr/local/ssl/lib -I$(CPP_PATH)/include -o $(MIX_APP_PATH)/priv/neutron_nif.so
+	  $(CC) $^ -shared -lei $(PLATFORM_OPTIONS) -fPIC -O3 -finline-functions -Wunused -Wall -Wpointer-arith -Wcast-align -Wcast-qual $(ERL_CFLAGS) $(ERTS_CFLAGS) $(ERL_LDFLAGS) -dynamiclib -pedantic -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib -I$(CPP_PATH)/include -o $(MIX_APP_PATH)/priv/neutron_nif.so
```

Comment:
might need to make this a dynamic var instead of hardcoded

Reply:
This needs to be moved into mac and linux special-casing

Reply:
ill just revert it. it just gives a warning